core/src/main/scala/org/apache/spark/SparkConf.scala

```diff
@@ -683,8 +683,9 @@ private[spark] object SparkConf extends Logging {
       AlternateConfig("spark.akka.frameSize", "1.6")),
     "spark.yarn.jars" -> Seq(
       AlternateConfig("spark.yarn.jar", "2.0")),
-    MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM.key -> Seq(
-      AlternateConfig("spark.reducer.maxReqSizeShuffleToMem", "2.3")),
+    NETWORK_MAX_REMOTE_BLOCK_SIZE_FETCH_TO_MEM.key -> Seq(
+      AlternateConfig("spark.reducer.maxReqSizeShuffleToMem", "2.3"),
+      AlternateConfig("spark.maxRemoteBlockSizeFetchToMem", "3.0")),
```

Comment:
will spark log warning message when these 2 alternatives are used? If not we still need to put them in `deprecatedConfigs`

Reply:
It will when user use old one, since deprecation warning also includes `allAlternatives`:
https://github.com/apache/spark/blob/77510c602a0141ac2bdf7576b5fbb38724637535/core/src/main/scala/org/apache/spark/SparkConf.scala#L753-L766

Reply:
I tested locally, Spark will log warning messages.